### PR TITLE
CB-14141 Remove jcenter() as it is not working (403) and is already obsoleted

### PR DESCRIPTION
**Pull request by bergerdenes**
See detailed description in the commit message.

### CODE REVIEW

**Review by foldik — APPROVED** [review]
Cool that someone drops out garbage.

File: core/build.gradle [hunk 1]
```diff
@@ -5,11 +5,9 @@ buildscript {
         maven { url = "$repoUrl" }
         maven { url = "$springRepoUrl" }
         mavenCentral()
-        jcenter()
     }
     dependencies {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
-        classpath "com.smokejumperit.gradle.license:Gradle-License-Report:0.0.2"
         classpath "gradle.plugin.com.github.jk1:gradle-license-report:0.3.4"
```
Comment (lajosrodek) [inline core/build.gradle:11]: What about this one? Is this possibly related to the one removed above?

Comment (bergerdenes) [reply core/build.gradle:11]: it is related to the jcenter removal (and it is used nowhere btw)

Comment (lajosrodek) [reply core/build.gradle:11]: Let me clarify. I was referring to this line:
```
classpath "gradle.plugin.com.github.jk1:gradle-license-report:0.3.4"
```
Should it be also removed in addition to line 12?

Comment (bergerdenes) [reply core/build.gradle:11]: it is used in core

File: .gitignore [hunk 2]
```diff
@@ -14,7 +14,7 @@ gradle
 *.iws
 .shelf/
 
-.vscode
```
Comment (lacikaaa) [inline .gitignore:17]: why is this removed?

Comment (bergerdenes) [reply .gitignore:17]: it was a duplicate of Line 8
